src/app/core/entity-details/entity-details/entity-details.component.ts

```diff
@@ -91,12 +96,25 @@ export class EntityDetailsComponent implements EntityDetailsConfig, OnChanges {
     }
     if (changes.id) {
       this.loadEntity(this.id);
-      // `initPanels()` is already called inside `loadEntity()`
+      this.subscribeToEntityChanges();
     } else if (changes.panels) {
+      // `initPanels()` is already called inside `loadEntity()`
```

Comment:
Why did you move this? I think at this place, the comment is a bit out of place: `initPanels` is called in this if statement, so what is this comment actually saying?

Reply:
sorry, I messed this up during merge conflicts. Thanks for pointing it out!